### ISSUE TITLE
Animation Panel: Direction radio input order for keyboard

### ISSUE THREAD

**Issue by BrittanyIRL** [GoogleForCreators/web-stories-wp#5613]
## Feature Description
There are several animation effects that allow a user to set direction of effect with a radio input. This feature works fine, however the keyboard usability is a little confusing because it's based off the index order of elements, the display isn't a standard list, it's a box with transformations applied to the list to create said box. We want the index order to read: ['top', 'right', 'bottom', 'left'], currently not all of them do this.

## Additional Context

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance Criteria

Keyboard usability of animation effect direction is clockwise.

**Comment by csossi** [comment]
Verified in QA

**Comment by ivayloklissarov** [comment]
Works fine, but it works backwards. It does not work properly with left and right arrow, only with up and down. Thus, I cannot select exactly the option that I want for instance.

**Comment by BrittanyIRL** [comment]
It's a little hairy, @ivayloklissarov because this is actually just a radio form input. So if we update how the DOM is handling this radio input vs others, while it appears wrong when we look at it, it's actually doing what it should do. That's my understanding of it. If we want to change that I suggest a separate ticket. 

![IMG_0264](https://user-images.githubusercontent.com/10720454/107082948-23f48880-67b2-11eb-9194-0c2c5f7646f8.jpg)

**Comment by ivayloklissarov** [comment]
Thanks for the explanation @BrittanyIRL ! All good then, nothing to worry about we talked about it with Sam also. Closing the ticket.
